### ISSUE TITLE
npc_text data from massparses 8.0.1, 8.1.0 & 8.1.5

### ISSUE THREAD

**Issue by mdX7** [TrinityCore/TrinityCore#23561]
**Description:**

Data from massparses of my bfa sniffs

8.0.1: https://gist.github.com/mdX7/a41ef0fecc51aa22cbbd7d54a16a315b
8.1.0: https://gist.github.com/mdX7/af72e6262b7f9c9cbeb83cc79e1eec24
8.1.5: https://gist.github.com/mdX7/0dd66f9a57c785fc797ab0f05fc9d02e

**Branch(es):**

master

**TC rev. hash/commit:** 

1bff903bd09a11865934796951a3905f5e8da748

**Comment by msoky** [comment]
ok to merge?

**Comment by illfated** [comment]
Looks like quite a few duplicates in the 3 files. Some of the IDs are updated at least 2 times in each file, as well as being updated to 3 different values in the first file.
Could they be filtered for duplicates in some easy way, or would that have to be done manually?

Out of curiosity, why update only the `VerifiedBuild` field without adding any data to the other fields in that row, if this is from sniffs?

**Comment by mdX7** [comment]
I actually parsed the files in 5 steps, I do not have enough RAM to load all of my sniffs into wpp at once (we're talking about a few hundred gigs of sniffs, yes i used filters). I parsed the first step, applied it to db, parsed 2nd step, apply [...]. Thats why fields are updated multiple times with different verifiedbuild.

VerifiedBuild only updates are as important as the others. That way we know that the data has not been updated compared to the already existing data in db.

We may just add this to the next tdb when funjoker or I bruteforced new stuff. Until then I'd also parse my other 815 and 820 sniffs and the 'duplicates' do not matter anymore. 
I won't remove them because I do not have time for this, if anyone wants to do it, give it a shot. It's just cosmetical anyways and may speedup execution of sql file by a few milliseconds.
